gtk4/src/auto/application.rs

```diff
@@ -301,6 +341,23 @@ pub trait GtkApplicationExt: IsA<Application> + 'static {
         }
     }
 
+    //#[doc(alias = "restore-state")]
```

Comment:
This signal lacks a Since annotation

Reply:
See https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/9053.